### PR TITLE
Serializer inheritance of attributes and relationships

### PR DESCRIPTION
**Pull request by aars**

### CODE REVIEW

**Comment by aars** [comment]
What conflicts?

**Comment by fotinakis** [comment]
screen shot 2016-02-01 at 1 35 38 pm

**Comment by aars** [comment]
Ugh. Sorry. Used an old master branch of yours was a bit confused when github didn't tell me what conflicts.

File: spec/serializer_spec.rb [hunk 1]
```diff
@@ -768,4 +768,36 @@ def serialize_primary(object, options = {})
       })
     end
   end
+
+  describe 'inheritance through subclassing' do
+    it 'inherits attributes' do
+      tagged_post = create(:tagged_post)
+      options = {serializer: MyApp::PostSerializerWithInheritedProperties}
+      data = JSONAPI::Serializer.serialize(tagged_post, options);
+      expect(data['data']['attributes']).to have_key('title');
```
Comment (fotinakis) [inline spec/serializer_spec.rb]: I'd rather this assert `eq(..)` to the actual content of the title, the factories create deterministic data for this reason.

Also, I you add the "tag" attribute to the subclass but I don't believe you test for it anywhere.

**Comment by fotinakis** [comment]
Back to you @aars

**Comment by aars** [comment]
@fotinakis You are right. Improved tests.

**Comment by fotinakis** [comment]
LGTM :+1: thanks!

**Comment by fotinakis** [comment]
Released in gem v0.6.0.

**Pull request merged**